src/process.c

```diff
@@ -287,6 +287,11 @@ cc_run_hook(struct oci_cfg_hook* hook, const gchar* state,
 		return false;
 	}
 
+	if (! hook->path) {
```

Comment:
@jodh-intel  I checked and there exists a test already to handle this. Looking further, the spec handler for hooks requires path to be present in the config json to complete successfully.
So this is a redundant check.